OMF_APITests/UnitTest1.cs

```diff
@@ -1,14 +1,249 @@
+            // Step 8 - Check Creations
+            Assert.True(checkCreations(sentData));
+            // Step 9 - Cleanup
+            Assert.True(cleanup());
+        }
+
+        private bool checkCreations(Dictionary<string, dynamic> sentData)
+        {
+            AppSettings settings = OMF_API.Program.getAppSettings();
+            IList<Endpoint> endpoints = settings.Endpoints;
+            dynamic omfTypes = OMF_API.Program.getJsonFile("OMF-Types.json");
+            dynamic omfContainers = OMF_API.Program.getJsonFile("OMF-Containers.json");
+            dynamic omfData = OMF_API.Program.getJsonFile("OMF-Data.json");
+
+            bool success = true;
+
+            foreach (var endpoint in endpoints)
+            {
+                try
+                {
+                    if (string.Equals(endpoint.EndpointType, "PI"))
+                    {
+                        // get point URLs
+                        HttpResponseMessage response = sendGetRequestToEndpoint(endpoint, $"{endpoint.BaseEndpoint}/dataservers?name={endpoint.DataServerName}").Result;
+                        string content = response.Content.ReadAsStringAsync().Result;
+                        dynamic dynamicJson = JsonConvert.DeserializeObject(content);
+                        string pointsURL = dynamicJson.Links.Points;
+
+                        // get point data and check response
+                        foreach (var omfContainer in omfContainers)
+                        {
+                            response = sendGetRequestToEndpoint(endpoint, $"{pointsURL}?nameFilter={omfContainer.id}*").Result;
+                            content = response.Content.ReadAsStringAsync().Result;
+                            dynamicJson = JsonConvert.DeserializeObject(content);
+
+                            // get end value URLs
+                            foreach (var item in dynamicJson.Items)
+                            {
+                                string endValueURL = item.Links.Value;
+                                // retrieve data
+                                response = sendGetRequestToEndpoint(endpoint, $"{endValueURL}").Result;
+                                content = response.Content.ReadAsStringAsync().Result;
+                                dynamicJson = JsonConvert.DeserializeObject(content);
+                                dynamic endValue = dynamicJson.Value;
+                                //check that the response was good and that data was written to the point
+                                JToken name = endValue.SelectToken("Name");
+                                if (!response.IsSuccessStatusCode)
+                                {
+                                    success = false;
+                                    Console.WriteLine($"Unable to find name {name}");
+                                }  
+                                else if (name != null && string.Equals(endValue.Name, "Pt Created"))
```

Comment:
Recommend using the `string.Equals()` overload that specifies a case insensitive `StringComparisonType`